File: strtolower/strtolower.js
// ATOMIC_FUNCTION:  strtolower(s)
// TAGS: string
// LANG:  node.js
// FUNCTION: strtolower(s)
function strtolower(s) {
    return s.toLowerCase();
}
var start_val = "HELLO";
var expected_end_val = "hello";
var end_val = strtolower(start_val);

// TEST
if(end_val==expected_end_val){
    console.log('PASS');
}else{
    console.log('FAIL');
}

File: strtolower/strtolower.java
public class strtolower {
    // ATOMIC_FUNCTION:  strtolower(s)
    // TAGS: string
    // LANG:  java
    // FUNCTION: strtolower(s)
    public String strtolower(String s){
        return s.toLowerCase();
    }

    public static void main(String args[]) {

        String start_val = "HELLO";
        String expected_end_val = "hello";
        strtolower a = new strtolower();
        String end_val = a.strtolower(start_val);

        // TEST
        if(end_val.equals(expected_end_val)) {
            System.out.println("PASS");
        } else {

            System.out.println("FAIL");
        }
    }
}
